### PR TITLE
struct shared module #3

### PR DESCRIPTION
**Pull request by walaayousef98**

### CODE REVIEW

File: ui/src/shared/shared-module/material-components-module/material-components-module.module.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    MatBottomSheetModule,
+    MatDialogModule,
+    MatSnackBarModule,
+    MatTooltipModule,
+    MatPaginatorModule,
+    MatSortModule,
+    MatTableModule
+  ]
+})
+export class MaterialComponentsModuleModule { }
```
Comment (diaahamad) [inline ui/src/shared/shared-module/material-components-module/material-components-module.module.ts:123]: module word here is duplicated

Comment (walaayousef98) [reply ui/src/shared/shared-module/material-components-module/material-components-module.module.ts]: Modified

**Comment by diaahamad** [comment]
read number 3 in the task again , material module should be separated in shared module

**Comment by walaayousef98** [comment]
check again

**Comment by diaahamad** [comment]
perfect

**Pull request merged**